debian/copyright

```diff
@@ -0,0 +1,30 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: stocks
+Upstream-Contact: Jan Veen <f1rst_unicorn@njsm.de>
```

Comment:
If you want, you can mention stocks@njsm.de as bug report contact, if debian has such a property.

Reply:
With your name or `The stocks developers`?

Reply:
Without my name, this will be processed by the ticket system.